apps/shipments/views/device.py

```diff
@@ -99,3 +104,36 @@ def telemetry(self, request, version, pk):
             telemetry_model_serializer.save()
 
         return Response(status=status.HTTP_204_NO_CONTENT)
+
+
+class SensorViewset(mixins.ListModelMixin, viewsets.GenericViewSet):
```

Comment:
This does not need to be a modelviewset if there is no model; I think this would be a little simpler to implement as an APIView.